src/mmw/mmw/settings/base.py

```diff
@@ -411,6 +411,17 @@ def get_env_setting(setting):
     'embed_flag': 'itsi_embed',
 }
 
+# HydroShare Integration Settings
+HYDROSHARE = {
+    'client_id': environ.get('MMW_HYDROSHARE_CLIENT_ID', 'model-my-watershed'),
+    'client_secret': environ.get('MMW_HYDROSHARE_SECRET_KEY',
```

Comment:
We can't have a default for this so setting the default value to a more explicit message may help with debugging a failure. 

```python
'client_secret': environ.get('MMW_HYDROSHARE_SECRET_KEY','MISSING MMW_HYDROSHARE_SECRET_KEY ENV VAR')
```

Reply:
Fixed here: https://github.com/WikiWatershed/model-my-watershed/blob/tt/hydroshare-export-login-backend/src/mmw/mmw/settings/base.py#L418